auxiliary/eus-multiple-values.l

```diff
@@ -0,0 +1,84 @@
+(make-package :mv-interface)
+(in-package :mv-interface)
+
+(defvar magical (cons 'multiple 'values))
+(defvar is-mv-on nil)
+
+(defun magicalp (x)
+  (and (consp x) (eq (car x) magical)))
+
+(defun list-values (&rest args)
+  "Takes a multiple-arguments return of values and make it a - simple - list"
+  (if (magicalp (car args))
+      (cdr (car args))
+      (car args)))
+
+(defun values (&rest args)
+  (if (or (and (not (null args))(null (cdr args)))
```

Comment:
Space after `))`.
`(not (null args))` -> `args` ?

Reply:
Oh, right. That seems silly now.